readthedocs/rtd_tests/tests/test_backend.py

```diff
@@ -78,6 +78,10 @@ def test_parse_git_tags(self):
                      self.project.vcs_repo().parse_tags(data)]
         self.assertEqual(expected_tags, given_ids)
 
+    def test_check_for_submodules(self):
+        repo = self.project.vcs_repo()
+        self.assertFalse(repo.submodules_exists())
```

Comment:
I'm not sure how to check when there is a submodule, should I create a submodule on other branch on the git repo setup and then check here?

Reply:
Actually I don't think I'm really testing here, I think this command can be only executed inside checkout.

Reply:
And I see on the logs that `make_test_git` says there is not git configuration (name and email) when running `    log.info(check_output(['git', 'add', '.'], env=env))`

This works on travis?